### ISSUE TITLE
Deprecation warnings

### ISSUE THREAD

**Issue by quozl** [sugarlabs/block-party-activity#29]
On Ubuntu 22.04 these warnings are shown

```log
% python3 BlockParty.py
BlockParty.py:86: DeprecationWarning: Gdk.Color.parse is deprecated
  colors = [Color(Gdk.Color.parse(i)[1]) for i in [
BlockParty.py:613: PyGTKDeprecationWarning: Using positional arguments with the GObject constructor has been deprecated. Please specify keyword(s) for "type" or use a class specific constructor. See: https://wiki.gnome.org/PyGObject/InitializerDeprecations
  win = Gtk.Window(Gtk.WindowType.TOPLEVEL)
BlockParty.py:144: DeprecationWarning: Gdk.Screen.get_width is deprecated
  self.window_w = self.window.get_screen().get_width()
BlockParty.py:145: DeprecationWarning: Gdk.Screen.get_height is deprecated
  self.window_h = self.window.get_screen().get_height() - gcs
BlockParty.py:53: DeprecationWarning: Gdk.Cursor.new is deprecated
  self._blank_cursor = Gdk.Cursor.new(Gdk.CursorType.BLANK_CURSOR)
```

**Comment by marsian83** [comment]
Hi @quozl  I have made a PR (#30) which addresses this directly

**Comment by quozl** [comment]
Thanks.

**Comment by chimosky** [comment]
Fixed in b91a70e.
